src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -1079,6 +1099,16 @@
       }
     }
   },
+  "coach-mark-title-font-size": {
+    "value": "{heading-size-xs}",
```

Comment:
@KayiuCarlson Looking at the token spec, I'm seeing the value is marked as `title-size-m`, could you double-check this?